tutor/src/screens/student-dashboard/cells.js

```diff
@@ -39,10 +46,14 @@ const ScoreCell = styled(Cell).withConfig({
   displayName: 'ScoreCell',
 })`
   padding-right: 1rem;
-  width: 10rem;
+  width: 8rem;
   ${({ theme }) => theme.breakpoint.mobile`
      flex: 1;
   `}
+
+  svg.fa-star {
+    padding: 3px;
+  }
 `;
```

Comment:
rather than using the `fa` class, how about using `ox-icon-star` that way it'd continue to work if we ever use other icons with Icon

Reply:
Yeah I was torn about that... this is trying to target bumping down the star size within the circle, so the specificity probably helps unexpected results later on.